conference/src/test/java/com/kyriba/conference/management/api/ScheduleRestControllerIntegrationTest.java

```diff
@@ -233,32 +191,31 @@ public void getPresentation_ifIncorrectIdResponseBadRequest() throws Exception
 
 
   @Test
-  public void removePresentationEndpointTest() throws Exception
+  public void removePresentationById() throws Exception
   {
     mockMvc.perform(delete("/api/v1/schedule/presentations/1000"))
         .andExpect(status().isNoContent())
         .andExpect(jsonPath("$").doesNotExist());
 
-    verify(scheduleService).deletePresentation(eq(1000L));
+    verify(scheduleService).deletePresentation(1000L);
   }
 
 
   @Test
-  public void removePresentation_ifIncorrectIdResponseBadRequest() throws Exception
+  public void ifIncorrectIdInDeletionResponseBadRequest() throws Exception
   {
-
     doThrow(mock(ConstraintViolationException.class))
         .when(scheduleService).deletePresentation(anyLong());
 
     mockMvc.perform(delete("/api/v1/schedule/presentations/-1000"))
         .andExpect(status().isBadRequest());
 
-    verify(scheduleService).deletePresentation(eq(-1000L));
+    verify(scheduleService).deletePresentation(-1000L);
   }
 
 
   @Test
-  public void getScheduleEndpointTest() throws Exception
+  public void getSchedule() throws Exception
```

Comment:
May be add suffix like HappyPath or Successfully or OK or isOk?

Reply:
done